### PR TITLE
python: Fix errors in processing TextMap

### PR DESCRIPTION
**Pull request by da-tanabe**
* Fix errors in processing TextMap.
* Support for SDK 0.13.18.
* Dropped support for pre-0.13.0 types.
* Pre-compile all DAML files used in the scope of tests (constantly rebuilding DAML files was taking an unacceptably long time).

### CODE REVIEW

File: python/dazl/damlsdk/package.py [hunk 1]
```diff
@@ -27,7 +27,10 @@ def package(options: 'PackageOptions', component: 'Optional[str]' = None) -> 'Pr
         raise ValueError(f'Unknown packaging component: {component}')
 
     LOG.info('Using %s to create a DAR package...', name)
-    args = [path / 'da-hs-damlc-app', *_damlc_package_options(options)]
+    jars = list(path.glob('*.jar'))
```
Comment (mschaef-da) [inline python/dazl/damlsdk/package.py]: What does this have to do with `TextMap`?

File: python/dazl/protocols/v1/pb_parse_event.py [hunk 2]
```diff
@@ -443,40 +445,29 @@ def to_variant(context: TypeEvaluationContext, tt: Type, variant: 'G.Variant') -
...
-            return to_natural_type(child_context.append_path('?'), ot.type_parameter, variant.value)
-
-    return type_evaluate_dispatch_default_error(
-        on_optional=process_optional,
-        on_map=process_map,
-        on_variant=process_variant)(context, tt)
+    return type_evaluate_dispatch_default_error(on_variant=process)(context, tt)
+
+
+def to_map(context: TypeEvaluationContext, tt: Type, map_pb: 'G.Map') -> Any:
```
Comment (mschaef-da) [inline python/dazl/protocols/v1/pb_parse_event.py]: Can we use a more specific return type? A function `to_map` that returns `Any` seems a bit disjoint to me.

File: python/dazl/damlsdk/package.py (hunk 1 above)
Comment (da-tanabe) [reply python/dazl/damlsdk/package.py]: The tests have been stuck on SDK 0.13.10 for a while because the mechanism used to fetch damlc had changed. This change allows me to go up to higher versions of the SDK.

File: python/dazl/protocols/v1/pb_parse_event.py (hunk 2 above)
Comment (da-tanabe) [reply python/dazl/protocols/v1/pb_parse_event.py]: Yep, fair…changed.
